### PR TITLE
Clean import-beats leftovers

### PR DESCRIPTION
**Pull request by mtojek**
Issue: https://github.com/elastic/package-registry/issues/414

This PR cleans the leftovers of the import-beats script.

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -68,18 +68,6 @@ func Build() error {
 	return sh.Run("go", "build", ".")
 }
 
-func ImportBeats() error {
```
Comment (ruflin) [inline magefile.go:71]: Line 35 also needs some cleanup.

Comment (mtojek) [reply magefile.go:71]: Fixed.

File: magefile.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ var (
 
 	publicDir    = "./public"
 	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/alpha/", "./dev/packages/example/", "./dev/packages/beats/"}
```
Comment (ruflin) [inline magefile.go:35]: Why did you remove the example ones too? I would keep them in as default but I opened https://github.com/elastic/package-registry/pull/435 to make it more flexible.

Comment (mtojek) [reply magefile.go:35]: Fixed.
